config/database.yml

```diff
@@ -19,6 +19,7 @@ default: &default
   encoding: unicode
   username: postgres
   password: e-ticket
+  port: 5432
```

Comment:
why you needed to add the port, was there an error ?

Reply:
@brahimdidi i had a problem with my Postgres i have to specify the port